src/glmfit.jl

```diff
@@ -229,6 +229,7 @@ abstract type AbstractGLM <: LinPredModel end
 mutable struct GeneralizedLinearModel{G<:GlmResp,L<:LinPred} <: AbstractGLM
     rr::G
     pp::L
+    f::Union{FormulaTerm,Nothing}
```

Comment:
can we call this `formula`? One of the hardest parts of understanding the GLM codebase is the huge number of overly succinct field names. Or maybe just add a comment here in the struct definition:

 ```suggestion
    f::Union{FormulaTerm,Nothing} # formula used to construct the model matrices
```